frontend/src/components/features/chat/event-message.tsx

```diff
@@ -48,19 +48,36 @@ export function EventMessage({
 
   const { data: config } = useConfig();
 
-  // Use our query hook to check if feedback exists and get rating/reason
   const {
     data: feedbackData = { exists: false },
     isLoading: isCheckingFeedback,
-  } = useFeedbackExists(isFinishAction(event) ? event.id : undefined);
+  } = useFeedbackExists(event.id);
+
+  const renderLikertScale = () => {
+    if (config?.APP_MODE !== "saas" || !isLastMessage || isCheckingFeedback) {
+      return null;
+    }
```

Comment:
For error observation, i want to ALWAYS show the feedback form if it is in the last 10 actions. @Openhands help me implement this

<img width="541" alt="image" src="https://github.com/user-attachments/assets/9e75ab1b-3228-4f2b-921f-82162ca61409" />